app/controllers/settings/profile_controller.rb

```diff
@@ -1,23 +1,24 @@
 # frozen_string_literal: true
 
-class Settings::ProfileController < Sellers::BaseController
+class Settings::ProfileController < Settings::BaseController
   before_action :authorize
 
   def show
     @title = "Settings"
-    @profile_presenter = ProfilePresenter.new(pundit_user:, seller: current_seller)
-    @settings_presenter = SettingsPresenter.new(pundit_user:)
+    profile_presenter = ProfilePresenter.new(pundit_user:, seller: current_seller)
 
-    @react_component_props = @settings_presenter.profile_props.merge(
-      @profile_presenter.profile_settings_props(request:)
+    render inertia: "Settings/Profile/Show", props: settings_presenter.profile_props.merge(
+      profile_presenter.profile_settings_props(request:)
     )
   end
 
   def update
```

Comment:
This method is used by both the profile setting page and the user profile pages so we migrate this with a conditional here which can be removed when we start migrating the user profile pages